data/config/packages.json

```diff
@@ -1648,6 +1649,12 @@
 			"key-commands": [
 				"sudo dpkg --add-architecture i386",
 				"sudo apt update"
+			],
+			"removekey-commands": [
+				"sudo apt autoremove lutris -y",
+				"sudo apt autoremove icoextract-thumbnailer -y",
+				"rm -r /home/*/.local/share/lutris",
```

Comment:
This looks all very dangerous  - so basically any account on the machine will have their local folders deleted.   An administrator of the account maybe just switching from one lutris package to another lutris based installation - so there wouldn't be any intent to break existing lutris info.